NOTEA/Controllers/HomeController.cs

```diff
@@ -47,16 +47,49 @@ public IActionResult CreateConspects(string date, string name, string conspectTe
 
             return View(model);
         }
-       
         public IActionResult CloseWindow()
         {
             TempData["SuccessMessage"] = "Your notea has been saved successfully!";
             return RedirectToAction(nameof(CreateConspects));
         }
+        public IActionResult UploadConspect()
+        {
+            return View(filemodel);
+        }
 
-        public IActionResult HowToUse()
+        [HttpPost]
+        public IActionResult UploadConspect(IFormFile file)
         {
-            
+            if(file.ContentType == "text/plain")
+            {
+                String buffer = "";
+                using (Stream stream = file.OpenReadStream())
+                {
+                    using (StreamReader sr = new StreamReader(stream))
+                    {
+                        while ((buffer = sr.ReadLine()) != null)
+                        {
+                            Console.WriteLine(buffer);
+                        }
+                    }
+                }
+                DataService.SaveConspects(
+                    new ConspectModel(
+                        "Remove this later",
```

Comment:
Remove this later?

Reply:
Čia data. Tiesiog placeholderis, kol pašalinsim. Nes susitarėm, kad datos mum nereik